OutOfSchool/OutOfSchool.IdentityServer/Data/Migrations/OutOfSchoolMigrations/20230203165048_AddIsBlockedColumnToWorkshop.cs

```diff
@@ -2,25 +2,24 @@
 
 #nullable disable
 
-namespace OutOfSchool.IdentityServer.Data.Migrations.OutOfSchoolMigrations
+namespace OutOfSchool.IdentityServer.Data.Migrations.OutOfSchoolMigrations;
+
+public partial class AddIsBlockedColumnToWorkshop : Migration
 {
-    public partial class AddIsBlockedColumnToWorkshop : Migration
+    protected override void Up(MigrationBuilder migrationBuilder)
     {
-        protected override void Up(MigrationBuilder migrationBuilder)
-        {
-            migrationBuilder.AddColumn<bool>(
-                name: "IsBlocked",
-                table: "Workshops",
-                type: "tinyint(1)",
-                nullable: false,
-                defaultValue: false);
-        }
+        migrationBuilder.AddColumn<bool>(
+            name: "IsBlocked",
+            table: "Workshops",
+            type: "tinyint(1)",
+            nullable: false,
+            defaultValue: false);
+    }
 
-        protected override void Down(MigrationBuilder migrationBuilder)
-        {
-            migrationBuilder.DropColumn(
-                name: "IsBlocked",
-                table: "Workshops");
-        }
+    protected override void Down(MigrationBuilder migrationBuilder)
+    {
+        migrationBuilder.DropColumn(
+            name: "IsBlocked",
+            table: "Workshops");
     }
-}
+}
```

Comment:
Migrations?????????? We already have them in develop branch. Maybe you have to exclude them from PR? ))))